ScheduleEvaluator/ConcreteCriterias/EnglishStart.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ScheduleEvaluator.ConcreteCriterias
+{
+    using Models;
+
+    class EnglishStart : Criteria
+    {
+        const int ENGLISH_DEPARTMENT = 28;
+        public EnglishStart(double weight) : base(weight)
+        {
+        }
+
+        public override double getResult(ScheduleModel s)
+        {
+            // This algorithm works for all of the 'preferred english starts' in
+            // the database right now. They are all apart of Department ID 28, my
+            // guess is that we can evaluate if a course is an english course by
+            // checking against Department ID. From my understanding this D-ID would
+            // change based on college.
+            foreach (Quarter q in s.Quarters) {
+                foreach (Course c in q.Courses) {
+                    if (c.DepartmentID == ENGLISH_DEPARTMENT) {
+                        return Int32.Parse(c.Description) == s.PreferenceSet.PreferredEnglishStart ?
+                            1.0 : 0.0;
```

Comment:
the goal of this criteria is to make sure english is scheduled in the first couple quarters right? Not to validate preferred english start. I mean that's a good criteria too, we just don't take that into account in scheduer